content/documentation/pages/3-stream-developer-guides/2-streams/3-data-flow-stream.md

```diff
@@ -51,124 +51,137 @@ Assuming Data Flow is [installed](%currentPath%/installation/) and running on on
 
 ### Application Registration
 
-Applications in Data Flow are registered as named resources so that they may be referenced when using the Data Flow DSL to configure and compose streaming pipelines. Registration associates a logical application name and type with a physical resource, given by a URI.
+Applications in Data Flow are registered as named resources so that they may be referenced when you use the Data Flow DSL to configure and compose streaming pipelines. Registration associates a logical application name and type with a physical resource, which is given by a URI.
```

Comment:
Yup. I debated with myself whether to change it everywhere, because it does get verbose. Really, though, we should change it to protect the brand and for consistency.